contracts/src/ERC20Destination.sol

```diff
@@ -85,25 +74,32 @@ contract ERC20Destination is IERC20Bridge, TeleporterTokenDestination, ERC20 {
     }
 
     /**
-     * @dev See {TeleportTokenDestination-_deposit}
+     * @dev See {TeleporterTokenDestination-_deposit}
      */
     function _deposit(uint256 amount) internal virtual override returns (uint256) {
         return SafeERC20TransferFrom.safeTransferFrom(this, amount);
     }
 
     /**
-     * @dev See {TeleportTokenDestination-_withdraw}
+     * @dev See {TeleporterTokenDestination-_withdraw}
      */
     function _withdraw(address recipient, uint256 amount) internal virtual override {
         _mint(recipient, amount);
     }
 
     /**
-     * @dev See {TeleportTokenDestination-_burn}
+     * @dev See {TeleporterTokenDestination-_burn}
      *
      * Calls {ERC20-_burn} to burn tokens from this contract.
      */
     function _burn(uint256 amount) internal virtual override {
         _burn(address(this), amount);
     }
+
+    /**
+     * @dev See {TeleporterTokenDestination-_scaleTokens}
+     */
+    function _scaleTokens(uint256 value, bool) internal pure override returns (uint256) {
+        return value;
+    }
```

Comment:
I'm not quite following why `_scaleTokens` needs to be a `virtual` function in `TeleporterTokenDestination` if it's just a no-op here.

Reply:
It basically stems from this conversation [here](https://github.com/ava-labs/teleporter-token-bridge/pull/49/files#r1552412169). Basically, it's so more code can be shared between the two destination contracts. We can also implement a multiplier in the ERC20 destination quite easily if we want to now.

Reply:
Cool, thanks for the context